scripts/verify-links.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# This script is used to verify links in markdown docs.
+#
+# - External links are ignored in CI because these links may go down out of our contorl.
+# - Anchors are ignored
+# - Internal links conventions
+#   - Links starting with '/media/' are relative to site root
+#   - Links starting with one or two dots are relative to the directory in which files reside
+#   - Other links are relative to the root of the version directory (e.g. v1.0, v2.0, v3.0)
+
+ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+cd $ROOT
+
+if ! which markdown-link-check &>/dev/null; then
+    sudo npm install -g markdown-link-check@3.7.3
+fi
+
+CONFIG_TMP=$(mktemp)
+
+trap 'rm -f $CONFIG_TMP' EXIT
+
+# TODO fix later
+IGNORE_DIRS=(v1.0 v2.0 v2.1 v2.1-legacy)
```

Comment:
there are some broken links in pre-v3.0 directories, but I don't want to make this PR too complicated. If this script is accepted, we can fix it later.